docs/dev-environments/ethereum/besu/docker-compose.yaml

```diff
@@ -66,18 +66,18 @@ services:
   # ethsigner2 is a 2nd signing wallet associated to node1, and provides an RPC endpoint for web3 APIs
   # The signing account is 0xb36b1934004385bfa5c51eaecb8ec348ec733ca8
   #
-  rpc.node2.avalon.org:
+  rpc.node2.avalon.local:
     image: pegasyseng/ethsigner:0.4.0
     command: >-
-      --chain-id=12345 --downstream-http-host=node2.avalon.org --downstream-http-port=8545 
+      --chain-id=12345 --downstream-http-host=node2.avalon.local --downstream-http-port=8545 
       --http-listen-host=0.0.0.0 --http-listen-port=8555 file-based-signer 
       --password-file=/var/lib/ethsigner/password.txt --key-file=/var/lib/ethsigner/keystore/account.txt
     ports:
       - 23001:8555
     volumes:
       - './ethsigner/node2:/var/lib/ethsigner'
     depends_on:
-      - node2.avalon.org
+      - node2.avalon.local
     networks:
       local_net:
         ipv4_address: '172.13.0.5'
```

Comment:
Can we remove static IPs in networks

Reply:
Facing issues when I try to remove static IPs. Truffle is not able to run the migration scripts. It hangs forever as suggested by logs-

truffle-envt-besu-setup |
truffle-envt-besu-setup | 1_initial_migration.js
truffle-envt-besu-setup | ======================
truffle-envt-besu-setup |
truffle-envt-besu-setup |    Deploying 'Migrations'
truffle-envt-besu-setup |    ----------------------
truffle-envt-besu-setup |    > transaction hash:    0x1a673f0f62c4f0690eb444d4922cd257b16f72bfff3e4d0362a50f3f9353ede6
truffle-envt-besu-setup | - Blocks: 0            Seconds: 0